src/loadlib.js

```diff
@@ -34,20 +34,37 @@ const AUXMARK       = [1];
 ** Returns the library; in case of error, returns NULL plus an
 ** error string in the stack.
 */
-const lsys_load = function(L, path) {
-    try {
-        path = lua.to_jsstring(path);
+let lsys_load;
+if (WEB) {
+    lsys_load = function(L, path) {
+        let xhr = new XMLHttpRequest();
+        xhr.open("GET", lua.to_jsstring(path), false);
+        xhr.send();
+        /* TODO: subresource integrity check? */
```

Comment:
No subresource integrity check is possible here (where would the subresource hash come from?). It only makes sense when we have a `<script>`.